packages/vscode-extension/src/commandsTreeViewProvider.ts

```diff
@@ -29,6 +29,11 @@ class TreeViewManager {
     const accountProvider = new CommandsTreeViewProvider([]);
     disposables.push(vscode.window.registerTreeDataProvider("teamsfx-accounts", accountProvider));
 
+    const environmentProvider = new CommandsTreeViewProvider([]);
```

Comment:
Does this view show for non-teamsfx project?

Reply:
No, already add check teamsfx project check logic, only show env in teamsfx project.